engine/js/BertaGallerySlideshow.js

```diff
@@ -16,15 +17,23 @@ var BertaGallerySlideshow = new Class({
       container.addClass('bt-is-mobile-device');
     }
 
-    this.attach(container);
+    this.getOptions();
+    this.attach();
     this.loadFirst();
   },
 
-  attach: function (container) {
-    this.container = container;
-    var fallbackGallery = this.container.getPrevious();
-    this.isRowFallback = fallbackGallery && fallbackGallery.hasClass('xGalleryType-row') ? true : false;
-    this.fullscreen = this.container.get('data-fullscreen') !== null;
+  getOptions: function () {
```

Comment:
Since you're not actually getting anything here I would suggest renaming it `initOptions`. Or you could use the return value - `this.options = getOptions()`